configure.ac

```diff
@@ -103,8 +98,12 @@ else
     fi
 fi
 
-# add user C_EXTRA_FLAGS back
-CFLAGS="$CFLAGS $USER_C_EXTRA_FLAGS"
+
+# Add Wall to CFLAGS
+if test "$GCC" = "yes"
+then
+    AM_CFLAGS="$AM_CFLAGS -Wall -Wno-unused"
```

Comment:
Any reason not to use `AX_HARDEN_CC_COMPILER_FLAGS`?

Reply:
Thanks @julek-wolfssl , using AX_HARDEN_CC_COMPILER_FLAGS is better! I had been looking at the wolfssl/configure.ac and did not realize that AX option added in more warnings/errors. No reason other than ignorance in this case, updating it.